### DISCUSSION TITLE
Command line arguments to change modes for scripting use

### DISCUSSION THREAD

**Discussion by EntityReborn** [seerge/g-helper#2259] [Ideas]
Allow for `ghelper.exe cpu turbo`, etc to allow for switching modes via scripting.
My use case is switching to Turbo when I launch Steam VR.

**Comment by seerge** [comment]
@EntityReborn hello, it’s not possible due to design of the app, it has to run in background to react on different system events, it can’t be restarted all the time.

To launch specific actions you can use hotkeys and emit them via autohotkey or any other automation software 

https://github.com/seerge/g-helper?tab=readme-ov-file#%EF%B8%8F-keybindings
